ambuda/views/proofing/main.py

```diff
@@ -184,7 +184,17 @@ def recent_changes():
     recent_revisions = (
         session.query(db.Revision).order_by(db.Revision.created.desc()).limit(100).all()
     )
-    return render_template("proofing/recent-changes.html", revisions=recent_revisions)
+    recent_activity = [("revision", r.created, r) for r in recent_revisions]
+
+    recent_projects = (
+        session.query(db.Project).order_by(db.Project.created_at.desc()).all()
```

Comment:
no `limit` on the projects pulled?

Reply:
Used 100 for consistency, but for the foreseeable future we'll have <100 projects